commit df87ecbf19109bab04a92df047a9949838206abc
Author: Per Forlin <per.forlin@linaro.org>
Date:   Fri Aug 19 14:52:36 2011 +0200

    fault-inject: export fault injection functions
    
    Export symbols should_fail() and fault_create_debugfs_attr() in order
    to let modules utilize the fault injection framework.
    
    Signed-off-by: Per Forlin <per.forlin@linaro.org>
    Acked-by: Akinobu Mita <akinobu.mita@gmail.com>
    Signed-off-by: Chris Ball <cjb@laptop.org>

diff --git a/lib/fault-inject.c b/lib/fault-inject.c
index f193b7796449..328d43357ed2 100644
--- a/lib/fault-inject.c
+++ b/lib/fault-inject.c
@@ -130,6 +130,7 @@ bool should_fail(struct fault_attr *attr, ssize_t size)
 
 	return true;
 }
+EXPORT_SYMBOL_GPL(should_fail);
 
 #ifdef CONFIG_FAULT_INJECTION_DEBUG_FS
 
@@ -243,5 +244,6 @@ struct dentry *fault_create_debugfs_attr(const char *name,
 
 	return ERR_PTR(-ENOMEM);
 }
+EXPORT_SYMBOL_GPL(fault_create_debugfs_attr);
 
 #endif /* CONFIG_FAULT_INJECTION_DEBUG_FS */